README.md

```diff
@@ -1,131 +1,37 @@
-# Loam SDK
+# Loam
 
-A Software Development Kit (SDK) and build tool for writing smart contracts in Rust on Wasm blockchains.
+Build **composable**, **upgradeable**, **secure** Smart Contracts. 
 
-Currently, the focus is on the Soroban VM, but the same ideas apply to other VMs.
+Build frontends that interact easily with (your own or anyone's) Smart Contracts.
 
-## Table of Contents
+Focus on the important parts of your [Soroban](https://soroban.stellar.org/) app. Let Loam handle the tedious bits.
 
-- [Loam SDK](#loam-sdk)
-  - [Table of Contents](#table-of-contents)
-  - [Getting Started](#getting-started)
-    - [Installation](#installation)
-    - [Setup](#setup)
-    - [Redeploy](#redeploy)
-  - [Subcontracts](#subcontracts)
-    - [Creating SubContracts](#creating-subcontracts)
-    - [External API](#external-api)
-  - [Core](#core)
-    - [Using the Core](#using-the-core)
+Loam is comprised of three main components:
+  1. Loam SDK - Create Smart Contracts for Soroban using smaller, more flexible building blocks called "Subcontracts".
+  2. Loam Frontend - Modern frontend tooling paired with a declarative environment configurations to help with local, test, and live blockchain networks.
+  3. Loam CLI - Build and deploy Loam Frontend to work with Smart Contracts.
```

Comment:
```suggestion
  2. Loam CLI - Build and deploy Loam Frontend to work with Smart Contracts.
  3. Loam Frontend - Modern frontend tooling paired with a declarative environment configurations to help with local, test, and live blockchain networks.

```

Reply:
I thought it made sense to mention the frontend before mentioning a CLI that helps work with the frontend